cli/up/build.go

```diff
@@ -87,22 +89,48 @@ func (cmd *up) buildImages(composeFile composeTypes.Config) (map[string]string,
 		images[svc.Name] = image{imageID, imageName}
 	}
 
-	// We expect each service to be sent exactly once, and we expect the channel
-	// to be closed when all images have been signalled as ready to push.
-	for base := range readyToPush {
-		if base.imageName != "" {
-			err = cmd.pushBaseTag(base.imageName, base.service)
+	var pp util.ProgressPrinter
+	progressPrinting := false
+loop:
+	for {
+		// If we are waiting for more than 5 seconds, print some indication of
```

Comment:
I wonder if 5 secs is too long, may even want it to be shorter, like 2 seconds?

Reply:
Can do.